routing/dht/dht.go

```diff
@@ -126,15 +125,7 @@ func (dht *IpfsDHT) putValueToPeer(ctx context.Context, p peer.ID,
 
 	pmes := pb.NewMessage(pb.Message_PUT_VALUE, string(key), 0)
 	pmes.Record = rec
-	rpmes, err := dht.sendRequest(ctx, p, pmes)
-	if err != nil {
-		return err
-	}
-
-	if !bytes.Equal(rpmes.GetRecord().Value, pmes.GetRecord().Value) {
-		return errors.New("value not put correctly")
-	}
-	return nil
```

Comment:
I can move this change to another PR, its more of a 'lower bandwidth usage' PR as opposed to the main goal of this PR